conda_smithy/templates/travis.yml.tmpl

```diff
@@ -9,6 +9,7 @@
 language: generic
 
 os: osx
+osx_image: xcode6.1
```

Comment:
I think we need to add a `beta-` here. Will see how PR ( https://github.com/conda-forge/staged-recipes/pull/1717 ) turns out and then update this accordingly.